include/simdjson/inline/document.h

```diff
@@ -322,7 +322,22 @@ inline bool document::dump_raw_tape(std::ostream &os) const noexcept {
 // parser inline implementation
 //
 really_inline parser::parser(size_t max_capacity) noexcept
-  : _max_capacity{max_capacity}, loaded_bytes(nullptr, &aligned_free_char) {}
+  :
```

Comment:
Please remove these changes to the constructor. It will handled in a distinct PR.

Reply:
done

Reply:
I still see the changes to the constructor... did you forget to commit the result or is GitHub giving me the wrong diff?

Reply:
going step by step. I'll commit when everything is finished.

Reply:
Ok. Got you.